app/resources/base.html.template

```diff
@@ -64,6 +66,9 @@
         {% endwith %}
       {% endif %}
     </style>
+    <!-- AMP Analytics -->
```

Comment:
Optional: It may be easier to read if you put both analytics implementations at the same place:

```
{% if env.amp %}
  <script async custom-element="amp-analytics"
              src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
{% else %}
  {% include "gtm.html.template" %}
{% endif %}
```

Reply:
I tried it but AMP Analytics require this position. @myoshimu can we move normal GTM tag to same place?

Reply:
Also I chatted with @myoshimu we need separate account for AMP GTM so this tag is essential for this purpose.